.gitignore

```diff
@@ -2,6 +2,7 @@
 
 # dependencies
 /node_modules
+/server/node_modules
```

Comment:
Do you have a node_modules in your local repository's server directory?...

Reply:
Haha yup, apparently... Not sure why!